src/main/java/org/jitsi/videobridge/IceUdpTransportManager.java

```diff
@@ -1445,7 +1445,7 @@ public BundleContext getBundleContext()
      * {@inheritDoc}
      */
     @Override
-    public DtlsControlImpl getDtlsControl(Channel channel)
+    public DtlsControlImpl getSrtpControl(Channel channel)
```

Comment:
that does make sense, but does any external caller of `IceUdpTransportManager::getSrtpControl` need to specifically know it's using a `DtlsControlImpl`?